src/features/inventory/provider.tsx

```
@@ -20,8 +21,22 @@ import {
 import { InventoryItemFactory } from './factories/InventoryItemFactory';
+import { useNotification } from '@/shared/hooks/useNotification';
 
 export function InventoryProvider({ children }: { children: ReactNode }) {
+  const { t } = useTranslation();
+  // Optional notification support - gracefully handle if NotificationProvider is not present
+  let showNotification: (
+    message: string,
+    variant?: 'success' | 'error' | 'info',
+  ) => void;
+  try {
+    const notification = useNotification();
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)